lib/src/Base/Func/InverseTrendTransform.cxx

```diff
@@ -68,37 +69,38 @@ InverseTrendTransform * InverseTrendTransform::clone() const
 /* Comparison operator */
 Bool InverseTrendTransform::operator ==(const InverseTrendTransform & other) const
 {
-  return (getEvaluation() == other.getEvaluation());
+  if (this == &other) return true;
+  return VertexValueFunction::operator==(other);
```

Comment:
This definition is useless, like `__str__` below, inheritance already does the job.

Reply:
right.